__tests__/__packages__/cli-test-utils/src/environment/doc/response/ITestEnvironment.ts

```diff
@@ -9,12 +9,23 @@
 *
 */
 
+import { AbstractSession } from "@zowe/imperative";
+import { ICommonJobParms, IJob } from "@zowe/zos-jobs-for-zowe-sdk";
+
 /**
  * The test environment for your test.
  * @export
  * @interface ITestEnvironment
  */
 export interface ITestEnvironment<TestPropertiesSchema> {
+    resources: {
```

Comment:
Could we add a small comment-block to document this new property on the inferface?
```suggestion
    /**
     * This property ...
     */
    resources: {
```